cli.js

```diff
@@ -41,6 +42,18 @@ const disableHooksParser = v => {
   }
 };
 
+const mapBaseUrlParser = v => {
+  const re = /(.*):(.*)/g;
+  let mapping = [];
+  if ((mapping = re.exec(v))===null || mapping.length!==3) {
+    throw new Error('Invalid --map-base-url flag. A mapping <url>:<baseFolder> with delimiter : expected.');
+  }
+  // folder is without trailing slash, so make sure that url has also no trailing slash:
+  mapBaseUrlToFolder.url = mapping[1].replace(/\/$/, '');
+  mapBaseUrlToFolder.folder = path.resolve(mapping[2]);
+  console.log(`mapping url ${mapBaseUrlToFolder.url} to folder ${mapBaseUrlToFolder.folder}`);
```

Comment:
```suggestion
  const mapping = v.split(':');
  if (mapping.length !== 2) {
    throw new Error('Invalid --map-base-url flag. A mapping <url>:<baseFolder> with delimiter : expected.');
  }
  // folder is without trailing slash, so make sure that url has also no trailing slash:
  mapBaseUrlToFolder.url = mapping[0].replace(/\/$/, '');
  mapBaseUrlToFolder.folder = path.resolve(mapping[1]);
  console.log(`mapping url ${mapBaseUrlToFolder.url} to folder ${mapBaseUrlToFolder.folder}`);
```

is much simpler.

Reply:
Thanks - I took over your suggestion!

Reply:
The regular expression is required, because the utl might have also a ':'.

Reply:
but as I understand, regular expression also will split parameter to 3 parts when someone pass url with `:`? Am I right, or not? 😅

Reply:
I wrote a simple test. The greedy nature of `.*` will make sure that we split correctly. Here an example:
  
``` js
   it('mapBaseUrlParser works with url mapping', async () => {
      const v = 'https:::::://schema.example.com/crm/:./test/docs/';
      const re = /(.*):(.*)/;
      const mapping = re.exec(v);
      expect(mapping.length).toBe(3);
      expect(mapping[0]).toBe(v);
      expect(mapping[1]).toBe('https:::::://schema.example.com/crm/');
      expect(mapping[2]).toBe('./test/docs/');
    });
```